ride-share.rb

```diff
@@ -0,0 +1,78 @@
+drivers = [
+  {id: "DR0004", trips: [ # driver
+    {date: "3rd Feb 2016", cost: 5, rider_id: "RD0022", rating: 5}, #trip
+    {date: "4th Feb 2016", cost: 10, rider_id: "RD0022", rating: 4}, # trip
```

Comment:
While I like the idea of this data structure, I might space it out in the file a little differently. In general, best practice is that unless the entire array or hash is going on one line, each key/value pair should get its own line. So in this case:

```ruby
drivers = [
  {
    id: "DR004",
    trips: [
      {date: "3rd Feb 2016", cost: 5, rider_id: "RD0022", rating: 5}, #trip
      {date: "4th Feb 2016", cost: 10, rider_id: "RD0022", rating: 4}, # trip
    ]
  }, {
    ... next driver
  }
}
```